src/lib/operators/abstract_aggregate_operator.cpp

```diff
@@ -29,24 +29,24 @@ const std::vector<std::shared_ptr<AggregateExpression>>& AbstractAggregateOperat
 const std::vector<ColumnID>& AbstractAggregateOperator::groupby_column_ids() const { return _groupby_column_ids; }
 
 std::string AbstractAggregateOperator::description(DescriptionMode description_mode) const {
-  const auto* const separator = description_mode == DescriptionMode::SingleLine ? " " : "\n";
+  const char separator = (description_mode == DescriptionMode::SingleLine ? ' ' : '\n');
 
   std::stringstream desc;
-  desc << AbstractOperator::description(description_mode) << separator << "GroupBy ColumnIDs: ";
+  desc << AbstractOperator::description(description_mode) << separator;
+  desc << "GroupBy ColumnIDs: {";
   for (size_t groupby_column_idx = 0; groupby_column_idx < _groupby_column_ids.size(); ++groupby_column_idx) {
     desc << _groupby_column_ids[groupby_column_idx];
```

Comment:
Sorry to bring that up in your PR. But why on earth don't we print the names of the group by columns? That has been bothering me for quite a long time.

We could extract the `column_name()` method of `abstract_join_operator.cpp:42` and use it here as well.

Reply:
Changed it:

<img width="397" alt="image" src="https://user-images.githubusercontent.com/9380917/131849462-fd2135b6-fd7e-4620-b08f-d9bded6acb42.png">


<img width="429" alt="image" src="https://user-images.githubusercontent.com/9380917/131849276-805cc9e2-ea93-4932-86fc-1d6e8a43de90.png">

 
<img width="505" alt="image" src="https://user-images.githubusercontent.com/9380917/131849318-002bceb3-8afc-4de2-bfd9-0cdd1cc15457.png">

Reply:
👍